analysis/data_utils.py

```diff
@@ -311,3 +315,87 @@ def experiment_level_ranking(experiment_snapshots_df,
     pivot_table = experiment_pivot_table(experiment_snapshots_df,
                                          benchmark_level_ranking_function)
     return experiment_level_ranking_function(pivot_table)
+
```

Comment:
consider moving these functions to a separate analysis/coverage_data_utils.py, as data_utils deals with the data coming from the DB, while these deal with data coming from the coverage JSON file.